src/System.IO.FileSystem/tests/FileStream/Dispose.cs

```diff
@@ -58,6 +63,92 @@ protected override void Dispose(bool disposing)
             }
         }
 
+
+        [Fact]
+        public void Dispose_ThrowDuringFlushWriteBuffer_TrueArgThrows()
+        {
+            RemoteInvoke(() =>
```

Comment:
Why is RemoteInvoke needed?

Reply:
Because the test is protecting regression of behavior that used to terminate the process. If that behavior regressed, we would like to get the test result.

Reply:
Ok.  FWIW, I think we may be using RemoteInvoke too much for such cases.  I suggest we should find an alternate way to be alerted to issues related to finalization; it's much broader than a specific test.